core/state_transition.go

```diff
@@ -76,7 +76,7 @@ type Message interface {
 }
 
 // IntrinsicGas computes the 'intrinsic gas' for a message with the given data.
-func IntrinsicGas(data []byte, contractCreation, homestead bool) (uint64, error) {
+func IntrinsicGas(data []byte, contractCreation, homestead bool, istanbul bool) (uint64, error) {
```

Comment:
It's nasty to pass a batch of fork indicators into the function. What about using a `dynasty`?